vhd/packer/install-dependencies.sh

```diff
@@ -270,6 +270,7 @@ for AZURE_CNI_NETWORKMONITOR_VERSION in ${AZURE_CNI_NETWORKMONITOR_VERSIONS}; do
 done
 
 AZURE_NPM_VERSIONS="
+1.0.33
```

Comment:
does NPM 1.0.33 require Azure CNI 1.0.33?

Reply:
nope, they're not related

I think Patrick updated CNI to v1.0.33, but not sure whether he did it for both windows and linux